accounts/forms.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+class SignUpForm(UserCreationForm):
+    class Meta:
+        model = User
+        fields = '__all__'
```

Comment:
ここは、
```suggestion
        fields = ('username', 'email', 'password1', 'password2')
```
としてください。__all__にすると、`migrations/0001_initial.py`のfields欄に記載されている要素全てを取得してしまいます。